src/sentry/utils/sdk.py

```diff
@@ -280,6 +280,9 @@ def _get_sdk_options() -> tuple[SdkConfig, Dsns]:
     sdk_options["release"] = (
         f"backend@{sdk_options['release']}" if "release" in sdk_options else None
     )
+    sdk_options.setdefault("_experiments", {}).update(
+        transport_http2=True,
+    )
```

Comment:
just to minimize risk a bit -- should the upgrade and enabling this be two separate patches? (would also allow fast reverting one without the other if it goes poorly)

Reply:
Yeah, makes sense. I was actually hoping to trigger this with a feature flag but not sure if _that_ makes sense.

Will update this to not have http2 turned on.

Reply:
@asottile-sentry reverted this part.